src/main/java/uk/gov/ons/census/notifyprocessor/utilities/TemplateMapper.java

```diff
@@ -8,14 +8,16 @@
 @Component
 public class TemplateMapper {
   private static final int HOUSEHOLD_ENGLAND = 1;
-  private static final int HOUSEHOLD_WALES = 2;
+  private static final int HOUSEHOLD_WALES_IN_ENGLISH = 2;
+  private static final int HOUSEHOLD_WALES_IN_WELSH = 3;
   private static final int HOUSEHOLD_NI = 4;
   private static final int INDIVIDUAL_RESPONSE_ENGLAND = 21;
-  private static final int INDIVIDUAL_RESPONSE_WALES = 22;
+  private static final int INDIVIDUAL_RESPONSE_WALES_IN_ENGLISH = 22;
```

Comment:
it's sent in welsh & english

Reply:
But the QID type refers to the type of questionnaire it launches, not the message it's distributed in

Reply:
I will change the template ID name to reflect that though, a good point I was not aware of